test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -258,6 +258,15 @@ def __init__(self, use_bias, postop, inplace_postop) -> None:
         def forward(self, x):
             return self.postop(self.linear(x))
 
+    class LinearUnaryModuleGelu(torch.nn.Module):
```

Comment:
Instead of creating a module of `LinearUnaryModuleGelu`, Can we modify and  reuse `LinearUnaryModule`?

Reply:
Yes, suggest to reuse `LinearUnaryModule` instead.